integration-tests/contract-set-delegate-auto-estimate.spec.ts

```diff
@@ -2,13 +2,13 @@ import { CONFIGS } from "./config";
 
 CONFIGS().forEach(({ lib, rpc, setup, knownBaker }) => {
   const Tezos = lib;
-  describe(`Test account delegation with estimation using: ${rpc}`, () => {
+  describe(`Test account delegation with estimation through contract api using: ${rpc}`, () => {
 
     beforeAll(async (done) => {
       await setup(true)
       done()
     })
-    it('delegates account to known baker with automatic estimate', async (done) => {
+    it('As a User I want to verify that I can delegate account to known baker with automatic estimate', async (done) => {
```

Comment:
As a User I want to verify -> Verify

Reply:
rewritten to  "Verify that an account can be delegated to known baker with automatic estimate"